source/elements/tags/Alert/Alert.example.jsx

```diff
@@ -0,0 +1,28 @@
+import Alert from './Alert';
+
+export default [{
+  name: 'Default Alert',
+  component: (
+    <Alert>Lorem ipsum</Alert>
```

Comment:
It looks like you may have intended this example to be labeled 'Default Example' in the HTML, and the below example to read 'Success alert'